.changeset/fifty-trees-do.md

```diff
@@ -0,0 +1,5 @@
+---
+'@rocket.chat/meteor': patch
+---
+
+Remove client side `starMessage` meteor method and replace this meteor method call with rest API.
```

Comment:
```suggestion
Removes client side `starMessage` meteor method and replaces this meteor method call with rest API.
```

But, I kinda think these changesets are not needed.

Or replace with something much much more generic, `Reduces websocket overhead from web client`.

given these are for customers, and not considered to be breaking, they do seem out of place in terms of friendliness and customer facing-ness (if that's a word);